packages/graphback-core/src/db/buildModelTableMap.ts

```diff
@@ -0,0 +1,68 @@
+import { GraphQLField, GraphQLFieldMap, GraphQLObjectType } from "graphql";
+import { parseDbAnnotations } from '../annotations/parser';
+import { defaultColumnNameTransform, defaultTableNameTransform } from './defaultNameTransforms';
+import { getPrimaryKey } from './getPrimaryKey';
+
+/**
+ * Contains mapping information between GraphQL Model type and database table
+ * 
+ * - typeName: Original GraphQLObjectType name
+ * - tableName: Name of datase table
+ * - id: Indicates the primary key field
+ * - fieldMap: Object of key-value mapping between GraphQL fields and database columns.
+ */
+export interface ModelTableMap {
+  typeName: string
+  tableName: string
+  idField: string
+  fieldMap?: any
```

Comment:
Maybe we can strongly type this to something like 
```suggestion
  fieldMap?: {[key:string]: string }
```